vcpkg-registry/versions/c-/cryptoutils.json

```diff
@@ -2,8 +2,8 @@
   "versions": [
     {
       "path": "$/ports/cryptoutils",
-      "version": "8.2.0",
-      "port-version": 0
+      "version": "8.3.0",
```

Comment:
Same question as above: Does this entry need to be updated manually for each Zilliqa release?

Reply:
Nope.